### ISSUE TITLE
Can't get it to work with local development pod

### ISSUE THREAD

**Issue by ExoticObjects** [neonichu/Rome#33]
I get this error:

```ruby
NoMethodError - undefined method integrate_targets= for #<Pod::Config:0x007fb4bb0b0aa0>
```

Here's my Podfile:
```ruby
source 'private-pod......-podspec.git'
source 'https://github.com/CocoaPods/Specs.git'

platform :ios, '9.0'
plugin 'cocoapods-rome'
use_frameworks!

def common_pods_for_target
  pod 'test_pod_2', :path => '../test_pod_2/'
end

target :test_use_frameworks_2 do
  common_pods_for_target
end
```

And here's my podspec:

```ruby
Pod::Spec.new do |s|
  s.name             = "test_pod_2"
  s.version          = "0.1.0"
  s.summary          = "Trying to get Cocoapods-Rome working" 
  s.description      = "Hopefully will build dynamic framework for me..."
  s.homepage         = "https://github.com/****/some_source"
  s.license          = 'MIT'
  s.author           = { "Author" => "blah@blah.com" }
  s.source           = {:git => "https://github.com/****/some_source.git"}
  s.platform     = :ios, '9.0'
  s.requires_arc = true
  s.source_files = 'Pod/Classes/**/*'
  s.resource_bundles = {
    'test_pod' => ['Pod/Assets/*.png']
  }
  s.dependency 'NGAParallaxMotion', '1.0.2'
  s.dependency 'Igor', '0.5.0'
  s.dependency 'UICKeyChainStore', '1.0.5'
  s.dependency 'AFNetworking', '2.5.2'
  s.dependency 'STHTTPRequest', '1.0.2'
  s.dependency 'SDWebImage'
  s.dependency 'SVProgressHUD', '1.0'
  s.dependency 'CocoaLumberjack/Swift'
  s.dependency 'ReactiveCocoa', '2.1.8'
  s.dependency 'BlocksKit', '2.2.5'
  s.dependency 'handlebars-objc', '1.4.3'
  s.dependency 'FontAwesome+iOS', '0.1.4'
  s.dependency 'FontAwesomeTools', '1.2'
  s.dependency 'ActiveRecord', '1.2.2'
  s.dependency 'UIView+AutoLayout'
  s.dependency 'FMDB', '2.6'
  s.dependency 'SimulatorStatusMagic', '1.7'

end
```

I'm using this command:

`pod _1.0.0.beta.2_ --verbose install`

This Podfile works fine when not using cocoapods-rome, so there are no issues there. 

Any ideas?

**Comment by segiddins** [comment]
I don't think Rome has been updated for the CocoaPods 1.0 beta yet.

**Comment by ExoticObjects** [comment]
That's true. Just tested without and it works. But the [Compatibility with CP 1.0 beta](https://github.com/neonichu/Rome/commit/0eb0a10d17f58e18d97b5d9152c63506251f0c33) commit led me to think otherwise, so I assumed it'd be ok.

**Comment by ExoticObjects** [comment]
Well, not so sure it's working, actually. I decided to try the dead-simple example in this project's readme:

```ruby
platform :ios, '9.0'

plugin 'cocoapods-rome'

pod 'Alamofire'
```
 and that fails.

Here are the steps I took:
- Made simple Xcode project
- Tried both `pod _0.39.0_ install` and simply `pod install`
- Here's console output:

```ruby
Analyzing dependencies
Downloading dependencies
Using Alamofire (3.1.5)
Generating Pods project
Sending stats
Building frameworks //FAILS AT THIS POINT
[!] /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -project Pods.xcodeproj -scheme Pods -configuration Release -sdk iphonesimulator -destination id=6150BC95-BE73-4E00-9752-E94492577D1B

Build settings from command line:
    SDKROOT = iphonesimulator9.2

2016-01-27 10:54:09.888 xcodebuild[87840:4904244] [MT] PluginLoading: Required plug-in compatibility UUID 9AFF134A-08DC-4096-8CEE-62A4BB123046 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/XToDo.xcplugin' not present in DVTPlugInCompatibilityUUIDs
...LOADS MORE ERRORS LIKE THIS
```

**Comment by neonichu** [comment]
IIRC, commit 0eb0a10d17f58e18d97b5d9152c63506251f0c33 was only for making Rome installable together with 1.0 beta, so that further testing can be done.

The `PluginLoading` errors are related to having third-party Xcode plugins installed which don't have the right compatibility UUIDs, that's completely unrelated to Rome. Do you see any build-related errors after those?

**Comment by ExoticObjects** [comment]
They're all plugin errors until this one:

```ruby
xcodebuild: error: Unable to find a destination matching the provided destination specifier:
		{ id:6150BC95-BE73-4E00-9752-E94492577D1B }

	Available destinations for the "Pods" scheme:
		{ platform:iOS Simulator, id:390A56B2-3ED5-45DA-9DEE-BF2593344C7B, OS:9.0, name:iPad 2 }
		{ platform:iOS Simulator, id:D789365D-057C-4B45-B3E9-57A62EF5CF78, OS:9.1, name:iPad 2 }
		{ platform:iOS Simulator, id:A96B63D0-C1C5-41F4-9AD8-31CF5D078493, OS:9.2, name:iPad 2 }
		{ platform:iOS Simulator, id:C2F62560-685E-4330-84E0-0A4292D82DD8, OS:9.0, name:iPad Air }
		{ platform:iOS Simulator, id:7305401B-B18D-47BB-817B-72C6FAF38023, OS:9.1, name:iPad Air }
		{ platform:iOS Simulator, id:8DBF4D27-E040-4EAF-A3EB-D3F3A40557B1, OS:9.2, name:iPad Air }
		{ platform:iOS Simulator, id:C581567F-BD76-4D93-87AB-991D0BBCA21B, OS:9.0, name:iPad Air 2 }
		{ platform:iOS Simulator, id:8C050D55-1CCF-4BA2-AB95-449D58508CE8, OS:9.1, name:iPad Air 2 }
		{ platform:iOS Simulator, id:94050BE2-F71A-40F2-B05A-C2F3C8E6BBB0, OS:9.2, name:iPad Air 2 }
		{ platform:iOS Simulator, id:99CE85C9-96B5-453C-8618-383A2090D0E1, OS:9.1, name:iPad Pro }
		{ platform:iOS Simulator, id:7D8F58DC-F4EE-4657-88B5-845AF28716A8, OS:9.2, name:iPad Pro }
		{ platform:iOS Simulator, id:2787B64A-C45B-44B5-B6E2-536A981E93A9, OS:9.0, name:iPad Retina }
		{ platform:iOS Simulator, id:B9BCACF3-B587-4016-83F8-BDB2B5C14804, OS:9.1, name:iPad Retina }
		{ platform:iOS Simulator, id:5012448F-9524-4AD4-A6C0-F1CEB2D6C4F9, OS:9.2, name:iPad Retina }
		{ platform:iOS Simulator, id:36ACD50A-FD84-43F7-ACD0-B6AD923985B2, OS:9.0, name:iPhone 4s }
		{ platform:iOS Simulator, id:52B52F1E-8AA9-4274-9BBB-BCAB01AF8E7C, OS:9.1, name:iPhone 4s }
		{ platform:iOS Simulator, id:3FFC3344-F788-4914-BB4C-F7D1CE4DA31A, OS:9.2, name:iPhone 4s }
		{ platform:iOS Simulator, id:7EECC2F5-4EF3-4D72-A81E-10CCDD6BEEB3, OS:9.0, name:iPhone 5 }
		{ platform:iOS Simulator, id:6FC4609D-0E21-479A-AE12-7F0AC5B6ECEF, OS:9.1, name:iPhone 5 }
		{ platform:iOS Simulator, id:30EAD8AB-43C6-4DF2-B7DB-1F7D2C3ACF94, OS:9.2, name:iPhone 5 }
		{ platform:iOS Simulator, id:C4C6980D-A8EA-4DA8-AEFD-EF886C631300, OS:9.0, name:iPhone 5s }
		{ platform:iOS Simulator, id:85789191-746F-44E2-9715-404471373D4A, OS:9.1, name:iPhone 5s }
		{ platform:iOS Simulator, id:9868A633-05C3-4E18-87C7-D95DE8CF18A0, OS:9.2, name:iPhone 5s }
		{ platform:iOS Simulator, id:82C336EF-39DD-4C35-B32B-87E1DB998E78, OS:9.0, name:iPhone 6 }
		{ platform:iOS Simulator, id:B930FBD2-1B80-400E-BCC8-1B95543957EF, OS:9.1, name:iPhone 6 }
		{ platform:iOS Simulator, id:59E045A8-3677-40BD-A014-13AD4384F983, OS:9.2, name:iPhone 6 }
		{ platform:iOS Simulator, id:8E80423A-DDDB-4C96-A991-8704CD68B348, OS:9.0, name:iPhone 6 Plus }
		{ platform:iOS Simulator, id:17856DBE-05B5-4BCA-972C-491C9F4C25E7, OS:9.1, name:iPhone 6 Plus }
		{ platform:iOS Simulator, id:CAB74A79-BFF2-48F8-8DEF-C1FA1CD595A1, OS:9.2, name:iPhone 6 Plus }
		{ platform:iOS Simulator, id:2131E03F-5B71-4F5B-8040-3FA5DF81A63B, OS:9.0, name:iPhone 6s }
		{ platform:iOS Simulator, id:9163E76C-F738-4719-A3AD-02FCF51CF1E1, OS:9.1, name:iPhone 6s }
		{ platform:iOS Simulator, id:31901D28-6F25-4DF9-A9C8-E6DC7A6941BC, OS:9.2, name:iPhone 6s }
		{ platform:iOS Simulator, id:8F4BD7BA-D458-45EA-8169-4CDA557E73BE, OS:9.0, name:iPhone 6s Plus }
		{ platform:iOS Simulator, id:93CB4783-AC32-41E6-AA86-6F5668945257, OS:9.1, name:iPhone 6s Plus }
		{ platform:iOS Simulator, id:01E317B3-D0E6-4ED8-893F-65931C282BEC, OS:9.2, name:iPhone 6s Plus }
```
What can I do about the plugin errors? Some of the plugins listed are things I don't care about, but I do care about others. And some seem vital, like:

```ruby
Support/Developer/Shared/Xcode/Plug-ins/CocoaPods.xcplugin
Support/Developer/Shared/Xcode/Plug-ins/NCSimulatorPlugin.xcplugin
Support/Developer/Shared/Xcode/Plug-ins/Open With Application.xcplugin
Support/Developer/Shared/Xcode/Plug-ins/Snapshots.xcplugin
```

**Comment by neonichu** [comment]
Interesting, seems like the simulator that has been picked cannot actually be used. Could you share the output of `xcrun simctl list` please?

**Comment by ExoticObjects** [comment]
```ruby
xcrun simctl list

== Device Types ==
iPhone 4s (com.apple.CoreSimulator.SimDeviceType.iPhone-4s)
iPhone 5 (com.apple.CoreSimulator.SimDeviceType.iPhone-5)
iPhone 5s (com.apple.CoreSimulator.SimDeviceType.iPhone-5s)
iPhone 6 (com.apple.CoreSimulator.SimDeviceType.iPhone-6)
iPhone 6 Plus (com.apple.CoreSimulator.SimDeviceType.iPhone-6-Plus)
iPhone 6s (com.apple.CoreSimulator.SimDeviceType.iPhone-6s)
iPhone 6s Plus (com.apple.CoreSimulator.SimDeviceType.iPhone-6s-Plus)
iPad 2 (com.apple.CoreSimulator.SimDeviceType.iPad-2)
iPad Retina (com.apple.CoreSimulator.SimDeviceType.iPad-Retina)
iPad Air (com.apple.CoreSimulator.SimDeviceType.iPad-Air)
iPad Air 2 (com.apple.CoreSimulator.SimDeviceType.iPad-Air-2)
iPad Pro (com.apple.CoreSimulator.SimDeviceType.iPad-Pro)
Apple TV 1080p (com.apple.CoreSimulator.SimDeviceType.Apple-TV-1080p)
Apple Watch - 38mm (com.apple.CoreSimulator.SimDeviceType.Apple-Watch-38mm)
Apple Watch - 42mm (com.apple.CoreSimulator.SimDeviceType.Apple-Watch-42mm)
== Runtimes ==
iOS 8.1 (8.1 - 12B411) (com.apple.CoreSimulator.SimRuntime.iOS-8-1)
iOS 8.2 (8.2 - 12D508) (com.apple.CoreSimulator.SimRuntime.iOS-8-2) (unavailable, failed to open liblaunch_sim.dylib)
iOS 8.3 (8.3 - 12F70) (com.apple.CoreSimulator.SimRuntime.iOS-8-3) (unavailable, failed to open liblaunch_sim.dylib)
iOS 9.0 (9.0 - 13A344) (com.apple.CoreSimulator.SimRuntime.iOS-9-0)
iOS 9.1 (9.1 - 13B143) (com.apple.CoreSimulator.SimRuntime.iOS-9-1)
iOS 9.2 (9.2 - 13C70) (com.apple.CoreSimulator.SimRuntime.iOS-9-2)
tvOS 9.1 (9.1 - 13U5077a) (com.apple.CoreSimulator.SimRuntime.tvOS-9-1)
watchOS 2.0 (2.0 - 13S343) (com.apple.CoreSimulator.SimRuntime.watchOS-2-0)
== Devices ==
-- iOS 8.1 --
    iPhone 4s (39E8C3F3-70CC-43A2-8517-E946B9932FE4) (Shutdown)
    iPhone 5 (5FC75C61-390C-4C6B-A336-1D990DA9EDA4) (Shutdown)
    iPhone 5s (6150BC95-BE73-4E00-9752-E94492577D1B) (Shutdown)
    iPhone 6 (001BA6CB-3F0B-4AF0-9295-86BCB68B629F) (Shutdown)
    iPhone 6 Plus (426FBE58-01FC-4E67-8274-B6DD0E84EE65) (Shutdown)
    iPad 2 (ED46F7D2-7202-4D8D-A8F6-23C37B838D7A) (Shutdown)
    iPad Retina (D7CED68E-8B2E-4CB3-B420-123F49DCDE7F) (Shutdown)
    iPad Air (CF1E7E9B-04D6-4297-A51B-23F975668074) (Shutdown)
    Resizable iPad (953F0E2E-20F3-4ACA-A1D5-7F46635F369E) (Shutdown) (unavailable, device type profile not found)
    Resizable iPhone (A94468A2-61DE-4216-979D-784C3BF9A003) (Shutdown) (unavailable, device type profile not found)
-- iOS 8.2 --
    iPhone 4s (A50909B5-3AA5-4C50-8645-ECADEE3D0FE7) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    iPhone 5 (A227FC86-AF24-4B9B-9F90-FAFC0BA28128) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    iPhone 5s (A1198C9C-21A4-40CA-BC1B-BDB6A1DC3D31) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    iPhone 6 (6F4F5688-E48D-48D2-9D16-1871DD370067) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    iPhone 6 Plus (2A781066-4245-4A40-88CD-945169BBC2E2) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    iPad 2 (0754B8FA-B5BE-4045-A6C3-A1F707EBFCF0) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    iPad Retina (545E52E8-1D42-4E3B-AC39-A0DBD6A9E39C) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    iPad Air (09F8C637-B05E-434D-BE2D-A1E43C50E27F) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    Resizable iPad (AB910A9A-D1BB-4179-A12D-60EAA9942F3A) (Shutdown) (unavailable, device type profile not found)
    Resizable iPhone (FCE60C8D-D64D-4C55-BBBD-9B206EE67288) (Shutdown) (unavailable, device type profile not found)
-- iOS 8.3 --
    iPhone 4s (6CF24755-637B-42B0-AE3C-40C77EEFA580) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    iPhone 5 (16801BB7-20F9-4859-BA68-C393FFE95B8D) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    iPhone 5s (5E547229-838D-46FF-91F9-46A12F4B11A1) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    iPhone 6 (ACA7CF51-76D6-4EE4-A571-B46FAC9DF9D8) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    iPhone 6 Plus (DA693EFC-9E1D-4E3C-8F5F-102F313A94FB) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    iPad 2 (E3608883-E426-4CB7-9241-982B2A6F58BB) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    iPad Retina (9EE7FED5-3B40-4007-8CA7-6A038EDAE8B5) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    iPad Air (586629B1-5E94-486E-AECB-520B79B43C5F) (Shutdown) (unavailable, failed to open liblaunch_sim.dylib)
    Resizable iPad (A5F8456F-5D0F-40BF-AD6C-30B4099A8AE8) (Shutdown) (unavailable, device type profile not found)
    Resizable iPhone (03B453C2-09F0-4CDB-B853-AAA7E64A9078) (Shutdown) (unavailable, device type profile not found)
-- iOS 9.0 --
    iPhone 4s (36ACD50A-FD84-43F7-ACD0-B6AD923985B2) (Shutdown)
    iPhone 5 (7EECC2F5-4EF3-4D72-A81E-10CCDD6BEEB3) (Shutdown)
    iPhone 5s (C4C6980D-A8EA-4DA8-AEFD-EF886C631300) (Shutdown)
    iPhone 6 (82C336EF-39DD-4C35-B32B-87E1DB998E78) (Shutdown)
    iPhone 6 Plus (8E80423A-DDDB-4C96-A991-8704CD68B348) (Shutdown)
    iPhone 6s (2131E03F-5B71-4F5B-8040-3FA5DF81A63B) (Shutdown)
    iPhone 6s Plus (8F4BD7BA-D458-45EA-8169-4CDA557E73BE) (Shutdown)
    iPad 2 (390A56B2-3ED5-45DA-9DEE-BF2593344C7B) (Shutdown)
    iPad Retina (2787B64A-C45B-44B5-B6E2-536A981E93A9) (Shutdown)
    iPad Air (C2F62560-685E-4330-84E0-0A4292D82DD8) (Shutdown)
    iPad Air 2 (C581567F-BD76-4D93-87AB-991D0BBCA21B) (Shutdown)
-- iOS 9.1 --
    iPhone 4s (52B52F1E-8AA9-4274-9BBB-BCAB01AF8E7C) (Shutdown)
    iPhone 5 (6FC4609D-0E21-479A-AE12-7F0AC5B6ECEF) (Shutdown)
    iPhone 5s (85789191-746F-44E2-9715-404471373D4A) (Shutdown)
    iPhone 6 (B930FBD2-1B80-400E-BCC8-1B95543957EF) (Shutdown)
    iPhone 6 Plus (17856DBE-05B5-4BCA-972C-491C9F4C25E7) (Shutdown)
    iPhone 6s (9163E76C-F738-4719-A3AD-02FCF51CF1E1) (Shutdown)
    iPhone 6s Plus (93CB4783-AC32-41E6-AA86-6F5668945257) (Shutdown)
    iPad 2 (D789365D-057C-4B45-B3E9-57A62EF5CF78) (Shutdown)
    iPad Retina (B9BCACF3-B587-4016-83F8-BDB2B5C14804) (Shutdown)
    iPad Air (7305401B-B18D-47BB-817B-72C6FAF38023) (Shutdown)
    iPad Air 2 (8C050D55-1CCF-4BA2-AB95-449D58508CE8) (Shutdown)
    iPad Pro (99CE85C9-96B5-453C-8618-383A2090D0E1) (Shutdown)
-- iOS 9.2 --
    iPhone 4s (3FFC3344-F788-4914-BB4C-F7D1CE4DA31A) (Shutdown)
    iPhone 5 (30EAD8AB-43C6-4DF2-B7DB-1F7D2C3ACF94) (Shutdown)
    iPhone 5s (9868A633-05C3-4E18-87C7-D95DE8CF18A0) (Shutdown)
    iPhone 6 (59E045A8-3677-40BD-A014-13AD4384F983) (Shutdown)
    iPhone 6 Plus (CAB74A79-BFF2-48F8-8DEF-C1FA1CD595A1) (Shutdown)
    iPhone 6s (31901D28-6F25-4DF9-A9C8-E6DC7A6941BC) (Shutdown)
    iPhone 6s Plus (01E317B3-D0E6-4ED8-893F-65931C282BEC) (Shutdown)
    iPad 2 (A96B63D0-C1C5-41F4-9AD8-31CF5D078493) (Shutdown)
    iPad Retina (5012448F-9524-4AD4-A6C0-F1CEB2D6C4F9) (Shutdown)
    iPad Air (8DBF4D27-E040-4EAF-A3EB-D3F3A40557B1) (Shutdown)
    iPad Air 2 (94050BE2-F71A-40F2-B05A-C2F3C8E6BBB0) (Shutdown)
    iPad Pro (7D8F58DC-F4EE-4657-88B5-845AF28716A8) (Shutdown)
-- tvOS 9.1 --
    Apple TV 1080p (9F389158-5BB2-4171-A6AA-847017EE4CC6) (Shutdown)
-- watchOS 2.0 --
    Apple Watch - 38mm (7C253F1B-3290-426A-AAD0-5301BD657BFA) (Shutdown)
    Apple Watch - 38mm (5E43AFC1-3DCC-4F69-8079-2449E91AE2FF) (Shutdown)
    Apple Watch - 42mm (5063EC67-7273-444D-A2B9-9CDA8E74E5F6) (Shutdown)
    Apple Watch - 42mm (9B629602-CFDC-488E-BF49-8492CD08BA42) (Shutdown)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.iOS-7-1 --
    iPhone 4s (BC813479-47B9-4AA6-B318-4624A626ACA1) (Shutdown) (unavailable, runtime profile not found)
    iPhone 5 (F199B402-6B6E-4E03-8517-F54005EDB88B) (Shutdown) (unavailable, runtime profile not found)
    iPhone 5s (6EB4584A-4392-457A-981C-BB085118BEA7) (Shutdown) (unavailable, runtime profile not found)
    iPad 2 (651C8FF2-B601-4458-9360-8D9EFBBBD629) (Shutdown) (unavailable, runtime profile not found)
    iPad Retina (85C34512-6145-4210-928A-0C7B628E54D1) (Shutdown) (unavailable, runtime profile not found)
    iPad Air (E8B5A8A0-2780-4AF3-85C0-B075E196DA0C) (Shutdown) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.iOS-8-0 --
    iPhone 4s (261DCF2B-D82A-4AB9-96AC-57CA914933FD) (Shutdown) (unavailable, runtime profile not found)
    iPhone 5 (E1A73E97-0FE7-49E4-BF61-BF8A5F120306) (Shutdown) (unavailable, runtime profile not found)
    iPhone 5s (1BEF95A9-6642-48B9-A940-7CA8F8A2051B) (Shutdown) (unavailable, runtime profile not found)
    iPhone 6 (657D5B6C-7BD6-4419-9382-9628D581E17A) (Shutdown) (unavailable, runtime profile not found)
    iPhone 6 Plus (200F62D2-6AAC-4A3A-8912-FE50D2A22D52) (Shutdown) (unavailable, runtime profile not found)
    iPad 2 (3B9A7D06-1289-43A2-8D61-4DF12780D8F3) (Shutdown) (unavailable, runtime profile not found)
    iPad Retina (3DF49BE5-C2C2-4738-B50A-1DE8DE94321A) (Shutdown) (unavailable, runtime profile not found)
    iPad Air (8CAF3A5F-DC21-4E1C-B5BC-280A3B5A5E8E) (Shutdown) (unavailable, runtime profile not found)
    Resizable iPad (B355AA81-B131-4557-9DB2-E07C650A21D9) (Shutdown) (unavailable, runtime profile not found)
    Resizable iPhone (248D3ACE-0514-4526-8A34-5561270E3428) (Shutdown) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.iOS-8-4 --
    iPhone 4s (E8785D45-61EE-4230-80A6-5FB7713EB4DC) (Shutdown) (unavailable, runtime profile not found)
    iPhone 5 (4BBE16CF-D716-4E6B-BA1E-01F77060C8C8) (Shutdown) (unavailable, runtime profile not found)
    iPhone 5s (9F93F1ED-A167-4A90-9307-1DD97CEB489C) (Shutdown) (unavailable, runtime profile not found)
    iPhone 6 (C3A6E2B1-0B49-40C7-827C-4C0525554339) (Shutdown) (unavailable, runtime profile not found)
    iPhone 6 Plus (8122D2C4-0046-44B5-853D-88A12B5104AE) (Shutdown) (unavailable, runtime profile not found)
    iPad 2 (C8587B1E-19AE-46E8-BF31-346947484E9A) (Shutdown) (unavailable, runtime profile not found)
    iPad Retina (1C16C650-F657-44B1-BBF8-1F2744A5F666) (Shutdown) (unavailable, runtime profile not found)
    iPad Air (461AED04-9132-469F-8532-875C1F0D3D59) (Shutdown) (unavailable, runtime profile not found)
    Resizable iPad (AE1191E5-C1BC-4E3A-AAC7-410CEDE47491) (Shutdown) (unavailable, runtime profile not found)
    Resizable iPhone (FBB8929F-E5B2-4B28-B4ED-F1232E512BD7) (Shutdown) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.iOS-9-3 --
    iPhone 4s (9CBDD031-2C8E-4F3B-AFE7-A070317C04E3) (Shutdown) (unavailable, runtime profile not found)
    iPhone 5 (B311A66E-4BCB-4E94-A5F0-8EB619231516) (Shutdown) (unavailable, runtime profile not found)
    iPhone 5s (1F53F6C8-8093-40F7-9377-0BDD26F8ADF7) (Shutdown) (unavailable, runtime profile not found)
    iPhone 6 (3B6FDA48-A016-4744-9F1D-988FC5060DEF) (Shutdown) (unavailable, runtime profile not found)
    iPhone 6 Plus (1EC35C0A-847C-4158-91CA-03EDD3F716D1) (Shutdown) (unavailable, runtime profile not found)
    iPhone 6s (20ACF134-0A80-420B-A833-663243CC20DD) (Shutdown) (unavailable, runtime profile not found)
    iPhone 6s Plus (FDFE1228-B1D6-4A69-AD8D-9ED8F3936998) (Shutdown) (unavailable, runtime profile not found)
    iPad 2 (1C7C09AD-9695-45AF-A01B-64B4CA14B3C9) (Shutdown) (unavailable, runtime profile not found)
    iPad Retina (CF2D785A-3D92-41E1-B550-803DB65B9336) (Shutdown) (unavailable, runtime profile not found)
    iPad Air (74F3E623-61B6-4555-A9EE-01877AED9460) (Shutdown) (unavailable, runtime profile not found)
    iPad Air 2 (2502B30F-55EE-4D18-9198-86D0BD515E39) (Shutdown) (unavailable, runtime profile not found)
    iPad Pro (AB8A6D8E-5A68-464C-98C0-5122A2466957) (Shutdown) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.tvOS-9-0 --
    Apple TV 1080p (B3D091C1-8B4E-4665-B674-36D1EF9EA464) (Shutdown) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.tvOS-9-2 --
    Apple TV 1080p (AA764E6E-4FE5-446E-8ECE-1CE22944FE92) (Shutdown) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.watchOS-2-2 --
    Apple Watch - 38mm (47480D51-BA88-4F3B-B9C3-A808048F7275) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch - 42mm (81F20071-84EB-4E3F-808D-4A6659F69557) (Shutdown) (unavailable, runtime profile not found)
== Device Pairs ==
B930D21F-B334-4578-B2A8-6022D1E2E772 (disconnected)
    Watch: Apple Watch - 38mm (7C253F1B-3290-426A-AAD0-5301BD657BFA) (Shutdown)
    Phone: iPhone 6 (82C336EF-39DD-4C35-B32B-87E1DB998E78) (Shutdown)
F90DD288-E8FE-4B56-822A-7A8A8BF642A8 (disconnected)
    Watch: Apple Watch - 38mm (5E43AFC1-3DCC-4F69-8079-2449E91AE2FF) (Shutdown)
    Phone: iPhone 6s (31901D28-6F25-4DF9-A9C8-E6DC7A6941BC) (Shutdown)
2298CD81-8123-470B-B2C9-60E78C513982 (disconnected)
    Watch: Apple Watch - 42mm (5063EC67-7273-444D-A2B9-9CDA8E74E5F6) (Shutdown)
    Phone: iPhone 6 Plus (8E80423A-DDDB-4C96-A991-8704CD68B348) (Shutdown)
6E28608E-869C-4640-B65C-423094865495 (disconnected)
    Watch: Apple Watch - 42mm (9B629602-CFDC-488E-BF49-8492CD08BA42) (Shutdown)
    Phone: iPhone 6s Plus (01E317B3-D0E6-4ED8-893F-65931C282BEC) (Shutdown)
E3DC429A-413A-4940-9A63-D3DBA9AE87A7 (unavailable)
    Watch: Apple Watch - 38mm (47480D51-BA88-4F3B-B9C3-A808048F7275) (Shutdown)
    Phone: iPhone 6s (20ACF134-0A80-420B-A833-663243CC20DD) (Shutdown)
409823CC-DAE0-40A8-B0A9-69E6DF48A921 (unavailable)
    Watch: Apple Watch - 42mm (81F20071-84EB-4E3F-808D-4A6659F69557) (Shutdown)
    Phone: iPhone 6s Plus (FDFE1228-B1D6-4A69-AD8D-9ED8F3936998) (Shutdown)
```

**Comment by neonichu** [comment]
Thanks, I see the issue now. Rome is not constraining the simulator search to the deployment target. This leads to it finding an 8.1 simulator in this case, but that is incompatible with the 9.0 deployment target.

**Comment by ExoticObjects** [comment]
Ok. Glad I was able to help find an actual bug.

Can you shed any light on the plugin errors discussed above?

**Comment by neonichu** [comment]
You can find some info about that here <http://buegling.com/blog/2014/6/5/xcode-6-plugins-and-you> — basically plugin authors have to update their plugins for each new version of Xcode or they won't get loaded.

**Comment by ExoticObjects** [comment]
Ok, but one of the plugins listed is cocoapods! 

And, I've built xcode projects on the command line before and not had this issue. Is Rome actually stopping the build because of these errors? (I can't know, as the simulator errors are definitely stopping my build). Odd that it would if `xcodebuild` doesn't.

**Comment by neonichu** [comment]
Ah, this is not CocoaPods the tool, but an Xcode plugin for it, possibly <https://github.com/kattrali/cocoapods-xcode-plugin>. Those errors won't affect the build.

**Comment by neonichu** [comment]
The actual bug with simulator search should be fixed by ac49307af83c8a5fc64fb4709b09a79da391a80e, thanks for reporting this, @ExoticObjects!

**Comment by neonichu** [comment]
Shipped as 0.4.2

**Comment by ExoticObjects** [comment]
Sweet. After running `gem update cocoapods-rome` this worked. Thanks!
